docs/serving/samples/knative-routing-go/README.md

```diff
@@ -21,6 +21,8 @@ the Login service.
    - In this example, we use `example.com`. If you don't have a domain name, you
      can modify your hosts file (on Mac or Linux) to map `example.com` to your
      cluster's ingress IP.
+   - If you have configured a custom domain for your Knative installation, we will
+     refer to it as <your domain name> in the rest of this document
```

Comment:
Throughout this file, replace <your domain name> with <YOUR_DOMAIN_NAME> to be consistent with the way other variables are referred to in this file.

Reply:
Done, thanks.